.github/workflows/debos.yml

```diff
@@ -95,4 +95,35 @@ jobs:
           # instruct fileserver to publish this directory
           url="${FILESERVER_URL}/${id}/"
           curl -X POST -H 'Accept: text/event-stream' "${url}"
+      - name: Prepare LAVA test job
+        run: |
+          id="${GITHUB_REPOSITORY}-${GITHUB_RUN_ID}-${GITHUB_RUN_ATTEMPT}"
+          url="${FILESERVER_URL}/${id}/"
+          BUILD_FILE_NAME=flash.tar.gz
+          BUILD_DOWNLOAD_URL="${url}${BUILD_FILE_NAME}"
+          LAVA_JOB_FILE="../ci/lava/rb3gen2-vision-kit-boot.yaml"
+
+          sed -i "s|{{BUILD_FILE_NAME}}|${BUILD_FILE_NAME}|g" ${LAVA_JOB_FILE}
+          sed -i "s|{{BUILD_DOWNLOAD_URL}}|${BUILD_DOWNLOAD_URL}|g" ${LAVA_JOB_FILE}
+          sed -i "s|{{GITHUB_SHA}}|${GITHUB_SHA}|g" ${LAVA_JOB_FILE}
+          sed -i "s|{{GITHUB_RUN_ID}}|${GITHUB_RUN_ID}|g" ${LAVA_JOB_FILE}
+          cat ${LAVA_JOB_FILE}
+          echo "LAVA_JOB_FILE=${LAVA_JOB_FILE}" >> "$GITHUB_ENV"
+
+      - name: Submit LAVA job
+        timeout-minutes: 30
+        uses: foundriesio/lava-action@v6
+        with:
+          lava_token: ${{ secrets.LAVATOKEN }}
+          lava_url: 'lava.infra.foundries.io'
+          job_definition: ${LAVA_JOB_FILE}
+          wait_for_job: true
+          fail_action_on_failure: false
```

Comment:
Mixed feelings on this; I guess it sucks to fail the whole build because we haven't managed to submit a test job to LAVA, but it's also synchronous (wait_for_job: true); makes me wonder if we should have this as a separate job as to be able to give just that part back